lib/constants.dart

```diff
@@ -2,4 +2,5 @@ class AppConstsnts {
   static const String endPoint = "https://demo.appwrite.io/v1";
   static const String project = "606d5bc9de604";
   static const String collection = "606d5bd9626ae";
+  static const String databaseID = "default";
```

Comment:
Let's just name this variable name to `database` to match the other IDs.

Reply:
How so? Like unique?